### DISCUSSION TITLE
Questions about plotting a graph showing accuracy improvement overtime

### DISCUSSION THREAD

**Discussion by AquifersBSIM** [chemprop/chemprop#1100] [Q&A] [answered]
Hello guys,

I am wondering if we can do the following in chemprop:
![image](https://github.com/user-attachments/assets/f5820bfc-df50-4421-bd5f-130909a94392)

I took the image from: https://medium.com/@mulugetas/drug-discovery-and-graph-neural-networks-gnns-a-regression-example-fc738e0f11f3
Essentially, we are showing how the prediction accuracy improve overtime.

Cheers,
Aquifer

**Comment by JacksonBurns** [comment, marked as answer]
Hi @AquifersBSIM Chemprop automatically logs all of its training progress to tensorboard (if installed with `pip install tensorboard`) - you can then see them with `tensorboard --logdir <YOUR_OUTPUT_DIRECTORY>`.

**Reply by AquifersBSIM** [reply]
Hi @JacksonBurns, thank you so much for the response, I am abit slow here! So that means I could actually plot a graph showing how accuracy improve across the epochs, by looking at the tensorboard?

**Reply by JacksonBurns** [reply]
> So that means I could actually plot a graph showing how accuracy improve across the epochs, by looking at the tensorboard?

Yes

**Reply by AquifersBSIM** [reply]
Hi @JacksonBurns , so I have enabled logger=True, and the output is lightning_logs, would this be correct?

```python
# Trainer
        trainer = Trainer(
            callbacks=[checkpointing],
            logger=True,
            enable_progress_bar=False,
            accelerator="gpu",
            devices=1,
            max_epochs=5,
            precision='16-mixed'
        )
```

**Reply by JacksonBurns** [reply]
Close, follow the example lightning has here: https://lightning.ai/docs/pytorch/stable/api/lightning.pytorch.loggers.tensorboard.html
